packages/wonder-blocks-button/components/button-core.js

```diff
@@ -36,19 +37,25 @@ export default class ButtonCore extends React.Component<Props> {
 
         const buttonStyles = _generateStyles(color, kind, light);
         const Tag = href ? "a" : "button";
+
+        const styleAttributes = processStyleList([
```

Comment:
I think we should create an HOC that we can use to wrap bare `<a>` and `<button>` tags within button-core.js and elsewhere so that the resulting component does this for us.  An HOC isn't a huge win over the current code, but in the future we'll want to wrap other things like `<input>` and maybe even `<svg>` etc.  Could you add a TODO?

Reply:
I'd rather use the HOC than changing the tag on `View`.  `View` now has some default styles on which we'd probably want to remove for other `tags`.